webpack.common.config.js

```diff
@@ -117,7 +117,12 @@ var resolve = {
         'highlight-css': nodePath('highlight.js/styles/default.css'),
         // Also alias some internal libraries for easy access
         'addons': path.join(__dirname, 'website', 'addons'),
-        'tests': staticPath('js/tests')
+        'tests': staticPath('js/tests'),
+        // GASP Items not defined as main in its package.json
+        'TweenLite' : nodePath('gsap/src/minified/TweenLite.min.js'),
+        'EasePack' : nodePath('gsap/src/minified/easing/EasePack.min.js'),
+        'YoutubeModal': staticPath('vendor/youtube.min.js')
```

Comment:
Ah, we should probably add `.min.js` here: https://github.com/CenterForOpenScience/osf.io/blob/f1f080736c958c524c5c16b18e35db22a3809fa2/webpack.common.config.js#L94